domains/lumidevelopment.json

```diff
@@ -0,0 +1,12 @@
+{
+  "owner": {
+    "username": "itsFatlum",
+    "email": "",
+    "discord": "563697359423406082"
+  },
+  "record": {
+    "MX": "mx1.improvmx.com.",
+    "MX": "mx2.improvmx.com.",
+    "TXT": "v=spf1 include:spf.improvmx.com ~all",
+  }
+}
```

Comment:
These file is completely out of wack. Recommend editing this file to reflect this:

```json
{
    "owner": {
        "username": "itsFatlum",
        "email": "",
        "discord": "563697359423406082"
    },
    "record": {
        "MX": [
            "mx1.improvmx.com",
            "mx2.improvmx.com"
        ],
        "TXT": [
            "v=spf1 include:spf.improvmx.com ~all"
        ]
    }
}
```